src/NBitcoin/Protocol/Versions.cs

```diff
@@ -75,6 +75,16 @@ public enum ProtocolVersion : uint
         /// <summary>
         /// shord-id-based block download starts with this version.
         /// </summary>
-        SHORT_IDS_BLOCKS_VERSION = 70014
+        SHORT_IDS_BLOCKS_VERSION = 70014,
+
+        /// <summary>
+        /// Oldest supported version of the CirrusNode which this node can connect to.
+        /// </summary>
+        CIRRUS_MIN_SUPPORTED_VERSION = 80000,
```

Comment:
I'm not a big fan of changing protocol version without actually changing the protocol, it kinda makes no sense. 
But I guess that's the easiest way to achieve the goal.

Reply:
In a way the protocol version should have been changed when we introduced additional payloads for PoA.
I agree this is not the intended use of the protocol version.